Plan/src/main/java/com/djrapitops/plan/system/info/connection/VelocityConnectionSystem.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * License is provided in the jar as LICENSE also here:
+ * https://github.com/Rsl1122/Plan-PlayerAnalytics/blob/master/Plan/src/main/resources/LICENSE
+ */
+package com.djrapitops.plan.system.info.connection;
+
+import com.djrapitops.plan.api.exceptions.connection.NoServersException;
+import com.djrapitops.plan.api.exceptions.database.DBOpException;
+import com.djrapitops.plan.system.database.databases.Database;
+import com.djrapitops.plan.system.info.InfoSystem;
+import com.djrapitops.plan.system.info.request.*;
+import com.djrapitops.plan.system.info.server.Server;
+import com.djrapitops.plan.system.info.server.ServerInfo;
+import com.djrapitops.plan.system.webserver.WebServerSystem;
+import com.djrapitops.plugin.api.TimeAmount;
+import com.djrapitops.plugin.api.utility.log.Log;
+
+import java.util.UUID;
+
+/**
+ * ConnectionSystem for Velocity.
+ *
+ * Based on BungeeConnectionSystem
+ *
+ * @author MicleBrick
+ */
+public class VelocityConnectionSystem extends ConnectionSystem {
```

Comment:
Duplication of this class is unnecessary, as only benefit is having it in velocity specific class.
This can be sorted later by refactoring common classes into network specific names